src/widgets/_buttons.scss

```
@@ -104,6 +104,39 @@ button {
         }
     }
 
+    &.back-button {
```

Is it dumb to leave a reminder here?

```suggestion
    // Stopgap since we can't do angled buttons in GtkCSS. Ideally we would 
    // deprecate this in favor of something like a Granite.BackButton with
    // custom drawing. Until then, stick a back icon into .back-button.
    &.back-button {
```